### PR TITLE
Update deploy cli

### PR DESCRIPTION
**Pull request by subodh101**
What?
- Remove setting common values at multiple places.
- Use `kpt` command directly to set values.

### CODE REVIEW

**Comment by subodh101** [comment]
[NOTE] Waiting on https://github.com/kubeflow/gcp-blueprints/pull/137, https://github.com/kubeflow/gcp-blueprints/pull/135

Do you think we should add optional parameters for `log-firewalls` and `gke.private` in the docs yet?
cc: @jlewi @Bobgy

File: content/en/docs/gke/deploy/deploy-cli.md [hunk 1]
```diff
@@ -112,19 +112,25 @@ kubeflow is defined. These are described in the table below.
 kpt setter | Description |
 -----------|-------------|
 mgmt-ctxt | This is the name of the KUBECONFIG context for the management cluster; this kubecontext will be used to create CNRM resources for your Kubeflow deployment. **The context must set the namespace to the namespace in your CNRM cluster where you are creating CNRM resources for the managed project.**|
-gcloud.core.project| The project you want to deploy in |
+name| This is the name of the Kubeflow deployment |
 location | The zone or region you want to deploy in |
-gcloud.compute.region | The region you are deploying in |
 gcloud.compute.zone | The zone to use for zonal resources; must be in gcloud.compute.region |
+gcloud.core.project| The project you want to deploy in |
+email| The email adderss of the admin |
```
Comment (8bitmp3) [inline content/en/docs/gke/deploy/deploy-cli.md:119]: Hey @subodh101 Can you check the spelling of "address" here pls? Thanks!

Comment (8bitmp3) [reply content/en/docs/gke/deploy/deploy-cli.md:119]: To improve formatting, can you also check that you have a blank line added before and after each code/command-line block please? Thank you!

**Comment by jlewi** [comment]
@subodh101 I think your changes applied to the master branch of kubeflow/gcp-blueprints and not the 1.1 instructions that this doc  is referring to e.g. in step 1

https://deploy-preview-2201--competent-brattain-de2d6d.netlify.app/docs/gke/deploy/deploy-cli/

```
kpt pkg get https://github.com/kubeflow/gcp-blueprints.git/kubeflow@v1.1.0 ./${KFDIR}
```

We probably want to cut v1.1.1 and then update the docs to point at it so the docs and Makefile changes are in sync.

**Comment by Bobgy** [comment]
/hold
Exactly what @jlewi said.

We can merge all 1.1.1 doc updates in a batch when we release it.

**Comment by Bobgy** [comment]
I just created https://github.com/kubeflow/gcp-blueprints/issues/139 to track related PRs, so we won't forget about them.

File: content/en/docs/gke/deploy/deploy-cli.md (hunk 1 above)
Comment (subodh101) [reply content/en/docs/gke/deploy/deploy-cli.md]: Thanks!

Comment (8bitmp3) [reply content/en/docs/gke/deploy/deploy-cli.md]: @subodh101 I noticed small thing - can you please change `bash` to `shell` in code blocks? I'm not sure if it makes a huge difference in terms of rendering on the site, but it's probably more proper. Thanks 👍 

```
    ```shell
    ...
    ```
```

**Comment by Bobgy** [comment]
@subodh101 shall we close this PR?

**Pull request closed without merging**